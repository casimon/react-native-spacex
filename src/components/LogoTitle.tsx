import React from 'react';
import {Image, StyleSheet} from 'react-native';

import images from '../images';

import colors from '../constants/colors';

const styles = StyleSheet.create({
  image: {
    width: 150,
    height: 30,
    tintColor: colors.white,
    flex: 1
  }
});

class LogoTitle extends React.Component {
  render() {
    return (
      <Image style={styles.image} source={images.logo} resizeMode="contain" />
    );
  }
}

export default LogoTitle;
