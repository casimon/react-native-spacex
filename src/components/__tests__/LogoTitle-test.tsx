/**
 * @format
 */

import 'react-native';
import React from 'react';
import LogoTitle from '../LogoTitle';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders logo title', () => {
  const tree = renderer.create(<LogoTitle />).toJSON();
  expect(tree).toMatchSnapshot();
});
