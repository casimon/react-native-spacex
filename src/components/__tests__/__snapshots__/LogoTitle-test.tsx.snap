// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders logo title 1`] = `
<Image
  resizeMode="contain"
  source={
    Object {
      "testUri": "../../../src/images/logo.png",
    }
  }
  style={
    Object {
      "flex": 1,
      "height": 30,
      "tintColor": "#ffffff",
      "width": 150,
    }
  }
/>
`;
